meta/runtime.yml

```diff
@@ -2,61 +2,113 @@ requires_ansible: '>=2.9.10'
 plugin_routing:
   modules:
     digital_ocean:
+      redirect: community.digitalocean.digitalocean
+    digitalocean:
       deprecation:
         removal_version: 2.0.0
         warning_text: see plugin documentation for details
     digital_ocean_account_facts:
```

Comment:
This bit doesn't look right, I *think* it should be
```yaml
digital_ocean_account_facts:
  redirect: community.digitalocean.account_facts
```

Then `account_facts` has the deprecation notice